Apps/WebClient/src/ClientApp/src/components/timeline/filters.vue

```diff
@@ -147,6 +159,13 @@ export default class FilterComponent extends Vue {
             return groups;
         }, []);
     }
+
+    private kFormatter(num: number) {
```

Comment:
Lets be explicit about the return type (string) since there is some math below
nit formatFilterCount

Reply:
done